src/cmd/mcp_route_test.go

```
@@ -0,0 +1,145 @@
+	"strings"
+	"testing"
+
+	uimcp "github.com/aldinokemal/go-whatsapp-web-multidevice/ui/mcp"
+	"github.com/gofiber/fiber/v3"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func mcpRPC(t *testing.T, app *fiber.App, body string, auth bool) (*httptest.ResponseRecorder, map[string]any) {
+	t.Helper()
+	req := httptest.NewRequest("POST", "/mcp", strings.NewReader(body))
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)